CMakeLists.txt

```diff
@@ -855,43 +863,62 @@ set(devilutionx_assets
   ui_art/mainmenuw.pcx
   ui_art/supportw.pcx)
 
-
-# Copy assets to the build assets subdirectory. This serves two purposes:
-# - If smpq is installed, devilutionx.mpq is built from these files.
-# - If smpq is not installed, the game will load the assets directly from this directoy.
-foreach(asset_file ${devilutionx_assets})
-  set(src "${CMAKE_CURRENT_SOURCE_DIR}/Packaging/resources/assets/${asset_file}")
-  set(dst "${DEVILUTIONX_ASSETS_OUTPUT_DIRECTORY}/${asset_file}")
-  list(APPEND DEVILUTIONX_MPQ_FILES "${asset_file}")
-  list(APPEND DEVILUTIONX_OUTPUT_ASSETS_FILES "${dst}")
-  add_custom_command(
-    COMMENT "Copying ${asset_file}"
-    OUTPUT "${dst}"
-    DEPENDS "${src}"
-    COMMAND ${CMAKE_COMMAND} -E copy "${src}" "${dst}"
-    VERBATIM)
-endforeach()
-if (Gettext_FOUND)
-  foreach(lang ${devilutionx_langs})
-    list(APPEND DEVILUTIONX_MPQ_FILES "${lang}.gmo")
-  endforeach()
+if(VIRTUAL_GAMEPAD)
+  list(APPEND devilutionx_assets
+    ui_art/button.png
+    ui_art/directions2.png
+    ui_art/directions.png
+    ui_art/menu-levelup.png
+    ui_art/menu.png)
 endif()
 
-if(BUILD_ASSETS_MPQ)
-  set(DEVILUTIONX_MPQ "${CMAKE_CURRENT_BINARY_DIR}/devilutionx.mpq")
-  add_custom_command(
-    COMMENT "Building devilutionx.mpq"
-    OUTPUT "${DEVILUTIONX_MPQ}"
-    COMMAND ${CMAKE_COMMAND} -E remove -f "${DEVILUTIONX_MPQ}"
-    COMMAND ${SMPQ} -M 1 -C PKWARE -c "${DEVILUTIONX_MPQ}" ${DEVILUTIONX_MPQ_FILES}
-    WORKING_DIRECTORY "${DEVILUTIONX_ASSETS_OUTPUT_DIRECTORY}"
-    DEPENDS ${DEVILUTIONX_OUTPUT_ASSETS_FILES} ${devilutionx_lang_targets} ${devilutionx_lang_files}
-    VERBATIM)
-  add_custom_target(devilutionx_mpq DEPENDS "${DEVILUTIONX_MPQ}")
-  add_dependencies(${BIN_TARGET} devilutionx_mpq)
+if(APPLE)
+  foreach(asset_file ${devilutionx_assets})
+    set(src "${CMAKE_CURRENT_SOURCE_DIR}/Packaging/resources/assets/${asset_file}")
+    get_filename_component(_asset_dir "${asset_file}" DIRECTORY)
+    set_source_files_properties("${src}" PROPERTIES
+      MACOSX_PACKAGE_LOCATION "Resources/assets/${_asset_dir}"

```

Comment:
Close:
```suggestion
      MACOSX_PACKAGE_LOCATION "Resources/${_asset_dir}"
```

Reply:
That would result in assets/assets/
See the build log here, the paths are already correct: https://github.com/diasurgical/devilutionX/runs/4423597286?check_suite_focus=true

Reply:
Ah, nvm, we *don't* want to place them under `assets/`

Reply:
Fixed

Reply:
> That would result in assets/assets/

Is it currently assets/assets/assets/?

The log says that they are in `Resources/assets/data/boxleftend.pcx`, but they should be in `Resources/data/boxleftend.pcx`